webcomponents/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vite";
+import { resolve } from "path";
+import dts from 'vite-plugin-dts'
+import cssInjectedByJsPlugin from 'vite-plugin-css-injected-by-js'
+
+export default defineConfig({
```

Comment:
Maybe it's better to add a `server` configuration to proxy WebSocket URL from Vite dev server to the actuall joyrtc server, just like `webui/vite.config.ts` does:

https://github.com/binbat/joyrtc/blob/fa06274932307d0605fb2be938107cac3288ee74/webui/vite.config.ts#L5-L13

or just merge such config into a public one